docs/source/prjxray/vtr-rrgraph.rst

```diff
@@ -329,6 +329,44 @@ Point Bag to CHANX / CHANY decomposition
    Currently this logic does not optimize for the lowest required
    track count, instead aiming to be correct first.
 
+A note on 7-series channel wires
+--------------------------------
+
+Most of the channel wires in the 7-series FPGAs are named using the following convention:
+
+DDLL,
+
+where DD is a direction in NN, SS, EE, WW, NE, NW, SE, SW, standing for
+north, south, east, west, northeast, northwest, southeast, and southwest, respectively,
+and LL is an integer corresponding to the Manhattan length of the wire in the number of tiles it spans.
+Wires with LL=1 occur twice as often as the others, and the two copies of each are designated as NR and NL for NN,
+SR and SL for SS, etc. For more details as well as exceptions to the above convention, please see the following `paper <https://infoscience.epfl.ch/record/282696?ln=en>`_.
```

Comment:
I wrote a Xilinx name decoder somewhere, let me see if I can find that. It would be awesome to resurrect that...

Reply:
Apparently -> https://github.com/SymbiFlow/symbiflow-arch-defs/issues/181

Reply:
There might be interesting information at https://docs.google.com/document/d/1KwERVU2Fpdgf1O0HCJT-Ri7kZCUzR_E4Aokn-LdTyPc/edit#